src/main/java/austin/Duke.java

```diff
@@ -0,0 +1,466 @@
+            fw.write((tasks.get(i)).toFileFormat());
+            i++;
+        }
+        fw.close();
+    }
+
+    /**
+     * Adds the specific task to the end of the text file.
+     * @param textToAdd New task added to the file for storage
+     * @throws IOException If there is an error due to file management
+     */
+    private static void appendToFile(String textToAdd) throws IOException {
+        FileWriter f = new FileWriter(FILE_PATH, true);
+        f.write(textToAdd);
+        f.close();
+    }
+
+    /**
+     * Prints the message once the command is called.
+     */
+    public static void printActivatedMessage() {
+        System.out.println("___________________COMMAND ACTIVATED__________________");
+    }
+
+    /**
+     * Prints the list of commands along with their description and format.
+     * This method is called once the "help" command is called.
+     *
+     * @param inputs Validate the command
+     * @throws AustinException If there are additional keywords
+     */
+    public static void printListOfCommands(String[] inputs) throws AustinException {
+        if (inputs.length > 1) {
+            throw new AustinException(INVALID_COMMAND_MESSAGE);
+        }
+        System.out.println("todo: Adds a todo task into the list.\n" +
+                "      Format: todo <task_description>\n");
+        System.out.println("event: Adds a event task into the list. " +
+                "The event date and time description is also needed " +
+                "while creating this task.\n" +
+                "      Format: event <task_description> | " +
+                "<date_and_time_information>\n");
+        System.out.println("deadline: Adds a task which has a deadline into the list. " +
+                "The deadline date and time information is also needed while " +
+                "creating this task.\n" +
+                "      Format: deadline <task_description> | " +
+                "<date_and_time_information>\n");
+        System.out.println("done: Marks a specific task as done.\n" +
+                "      Format: done <task_id>\n");
+        System.out.println("list: Prints all the tasks along with the status of" +
+                " each task.\n" +
+                "      Format: list\n");
+        System.out.println("delete: Removes a task from the list.\n" +
+                "      Format: delete <task_id>\n");
+        System.out.println("bye: Exits the program with a goodbye message.\n" +
+                "      Format: bye\n");
+        System.out.println("help: Prints a list of commands.\n" +
+                "      Format: help");
+    }
+
+    /**
+     * Prints after the command is successfully executed.
+     */
+    public static void printCompletionMessage() {
+        System.out.println("___________________COMMAND EXECUTED___________________");
+        System.out.println("Anything else?\n"
+                + "In case, if you are unsure of any commands, please type \"help\".");
+    }
+}
```

Comment:
Most of your methods are packed into this one class. Please consider placing some of the methods in other classes.